commcare_connect/templates/account/base.tailwind.html

```diff
@@ -0,0 +1,57 @@
+{% load static i18n %}
+
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>{% block title %}CommCareConnect - {% block head_title %}{% endblock head_title %}{% endblock title %}</title>
+    {% load static %}
+    {% block css %}
+      <link rel="stylesheet" href="{% static 'bundles/css/tailwind.css' %}">
+      <link href="https://fonts.googleapis.com/css2?family=Work+Sans:wght@400;500;600;700&display=swap" rel="stylesheet">
+      <link rel="stylesheet" href="https://cdn.jsdelivr.net/gh/aquawolf04/font-awesome-pro@5cd1511/css/all.css" />
+    {% endblock %}
+    {% block javascript %}
+      <script src="{% static 'bundles/js/vendors-bundle.js' %}" defer></script>
+      <script src="{% static 'bundles/js/tailwind-bundle.js' %}" defer></script>
+    {% endblock javascript %}
```

Comment:
We should extend this from the tailwind/base.html, so that we don't repeat all these imports here.

Reply:
That will require changing the tailwind/base.html as it has top level template imports for headers and sidenav which are not needed on this page.